ios/Exponent/Kernel/DevSupport/EXHomeModule.m

```diff
@@ -21,6 +21,144 @@ @implementation EXHomeModule
 
 + (NSString *)moduleName { return @"ExponentKernel"; }
 
+/** embedded.mobileprovision plist format:
+ 
+ AppIDName, // string — TextDetective
+ ApplicationIdentifierPrefix[],  // [ string - 66PK3K3KEV ]
+ CreationData, // date — 2013-01-17T14:18:05Z
+ DeveloperCertificates[], // [ data ]
+ Entitlements {
+ application-identifier // string - 66PK3K3KEV.com.blindsight.textdetective
+ get-task-allow // true or false
+ keychain-access-groups[] // [ string - 66PK3K3KEV.* ]
+ },
+ ExpirationDate, // date — 2014-01-17T14:18:05Z
+ Name, // string — Barrierefreikommunizieren (name assigned to the provisioning profile used)
+ ProvisionedDevices[], // [ string.... ]
+ TeamIdentifier[], // [string — HHBT96X2EX ]
+ TeamName, // string — The Blindsight Corporation
+ TimeToLive, // integer - 365
+ UUID, // string — 79F37E8E-CC8D-4819-8C13-A678479211CE
+ Version, // integer — 1
+ ProvisionsAllDevices // true or false  ***NB: not sure if this is where this is
+ 
+ */
+
+-(NSDictionary*) getMobileProvision {
+  static NSDictionary* mobileProvision = nil;
```

Comment:
since this method is an instance method, i'd use a property in the class to save the value instead of making a static variable. you can see an example of this here https://github.com/expo/expo/blob/5bf4c3d5071cf72469fb41e3dc71ff93d3c5ff79/ios/Exponent/Kernel/AppLoader/CachedResource/EXManifestResource.m#L185-L219 (property defined on L23)